src/renderer/store/modules/playlists.js

```diff
@@ -334,18 +360,46 @@ const actions = {
     }
   },
 
-  async removePlaylist({ commit }, playlistId) {
+  async removePlaylist({ state, commit, rootState, dispatch }, playlistId) {
     try {
       await DBPlaylistHandlers.delete(playlistId)
+
+      const quickBookmarkPlaylistWasDeleted = playlistId === rootState.settings.quickBookmarkTargetPlaylistId
+      if (quickBookmarkPlaylistWasDeleted) {
+        // TODO: use user's top sorting preference, or most recently used playlist.
+        // The sorting logic is currently not in an accessible utility class.
+        const chosenPlaylist = state.playlists.find((playlist) => playlist._id !== playlistId)
+        if (chosenPlaylist) {
+          dispatch('updateQuickBookmarkTargetPlaylistId', chosenPlaylist._id, { root: true })
+          showToast(i18n.t('User Playlists.SinglePlaylistView.Toast["Playlist {playlistName} is the new quick bookmark playlist."]', {
+            playlistName: chosenPlaylist.playlistName
+          }))
+        }
+      }
+
       commit('removePlaylist', playlistId)
     } catch (errMessage) {
       console.error(errMessage)
     }
   },
 
-  async removePlaylists({ commit }, playlistIds) {
+  async removePlaylists({ state, commit, rootState, dispatch }, playlistIds) {
     try {
       await DBPlaylistHandlers.deleteMultiple(playlistIds)
+
+      const quickBookmarkPlaylistWasDeleted = playlistIds.indexOf(rootState.settings.quickBookmarkTargetPlaylistId) !== -1
```

Comment:
```suggestion
      const quickBookmarkPlaylistWasDeleted = playlistIds.includes(rootState.settings.quickBookmarkTargetPlaylistId)
```

Reply:
As to why I chose `indexOf` instead of `includes`, I found some research indicating that it's more performant on almost every browser, but I agree that it's right to change these because it does ultimately make for cleaner and more readable code.